app/entry.server.tsx

```diff
@@ -0,0 +1,35 @@
+import { renderToString } from 'react-dom/server';
+import createCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/react';
+import createEmotionServer from '@emotion/server/create-instance';
+import type { EntryContext } from '@remix-run/node';
+import { RemixServer } from '@remix-run/react';
+
+const key = 'custom';
+const cache = createCache({ key });
+const { extractCriticalToChunks, constructStyleTagsFromChunks } = createEmotionServer(cache);
+
+export default function handleRequest(
+  request: Request,
+  responseStatusCode: number,
+  responseHeaders: Headers,
+  remixContext: EntryContext,
+) {
+  let markup = renderToString(
+    <CacheProvider value={cache}>
+      <RemixServer context={remixContext} url={request.url} />
+    </CacheProvider>,
+  );
+
+  const chunks = extractCriticalToChunks(markup);
+  const styles = constructStyleTagsFromChunks(chunks);
+
+  markup = markup.replace('__STYLES__', styles);
```

Comment:
Nødvendig for ikke å få hydreringsfeil ved bruk av `@emotion`: https://bensthoughts.dev/blog/remix/remix-emotion-styled